packages/docusaurus-theme-classic/src/theme/SiteMetadata/index.tsx

```diff
@@ -24,10 +24,18 @@ import SearchMetadata from '@theme/SearchMetadata';
 // See https://github.com/facebook/docusaurus/issues/3317
 function AlternateLangHeaders(): JSX.Element {
   const {
-    i18n: {defaultLocale, localeConfigs},
+    i18n: {currentLocale, defaultLocale, localeConfigs},
   } = useDocusaurusContext();
   const alternatePageUtils = useAlternatePageUtils();
 
+  const currentHtmlLang = localeConfigs[currentLocale]?.htmlLang;
+
+  // HTML lang is a BCP 47 tag, but the Open Graph protocol requires
+  // using underscores instead of dashes.
+  // See https://ogp.me/#optional
+  // See https://en.wikipedia.org/wiki/IETF_language_tag)
+  const bcp47ToLocale = (code: string): string => code.replace('-', '_');
```

Comment:
```suggestion
  const bcp47ToOpenGraphLocale = (code: string): string => code.replace('-', '_');
```

More explicit fn name